vueapp/src/components/Game.vue

```diff
@@ -28,6 +28,7 @@
       <v-card-text> TODO </v-card-text>
       <v-card-actions>
         <v-btn text="REMOVE" @click="addGameFromCourtToQueue()"></v-btn>
+        <v-btn text="COMPLETE GAME" @click="completeGame(court)"></v-btn>
```

Comment:
You don't really need to pass the `court` object to the function since there can only be one court in this vue.
```suggestion
        <v-btn text="COMPLETE GAME" @click="completeGame"></v-btn>
```